server/src/main/java/io/unitycatalog/server/persist/converters/TableInfoConverter.java

```diff
@@ -94,7 +92,6 @@ public static TableInfoDAO convertToDAO(TableInfo dto) {
                 .url(dto.getStorageLocation() != null ? dto.getStorageLocation() : null)
                 .type(dto.getTableType().toString())
                 .dataSourceFormat(dto.getDataSourceFormat().toString())
-                .url(dto.getStorageLocation())
```

Comment:
@alexreid-db Isn't `dto.getStorageLocation() != null ? dto.getStorageLocation() : null` equivalent to `dto.getStorageLocation()`? So, shouldn't this line be left as is and line 94 be removed?

Reply:
yeah, sorry, didn't look at the line above closely enough